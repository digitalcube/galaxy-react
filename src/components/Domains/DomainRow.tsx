import React, { FC, PropsWithChildren } from 'react';

export type DomainRowProps = PropsWithChildren<{
  className?: string;
}>;
export const DomainRow: FC<DomainRowProps> = ({ className, children }) => {
  return (
    <div className={`live-domain${className ? ` ${className}` : ''}`}>
      {children}
    </div>
  );
};
