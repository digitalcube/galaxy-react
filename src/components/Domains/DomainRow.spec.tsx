import { render } from '@testing-library/react';
import React from 'react';
import { DomainRow } from './DomainRow';

describe('DomainRow', () => {
  it('to match snapshot', () => {
    const { asFragment } = render(
      <DomainRow>
        <a href="#">Item</a>
      </DomainRow>
    );
    expect(asFragment()).toMatchSnapshot();
  });
  it('to match snapshot [full props]', () => {
    const { asFragment } = render(
      <DomainRow className="test">
        <a href="#">Item</a>
      </DomainRow>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
