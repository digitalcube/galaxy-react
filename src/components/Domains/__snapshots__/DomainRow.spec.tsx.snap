// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DomainRow to match snapshot [full props] 1`] = `
<DocumentFragment>
  <div
    class="live-domain test"
  >
    <a
      href="#"
    >
      Item
    </a>
  </div>
</DocumentFragment>
`;

exports[`DomainRow to match snapshot 1`] = `
<DocumentFragment>
  <div
    class="live-domain"
  >
    <a
      href="#"
    >
      Item
    </a>
  </div>
</DocumentFragment>
`;
